src/Monomer/Widgets/Animation/Fade.hs

```diff
@@ -50,54 +41,43 @@ Configuration options for fade:
 - 'autoStart': whether the first time the widget is added, animation should run.
 - 'duration': how long the animation lasts in ms.
 - 'onFinished': event to raise when animation is complete.
+- 'onFinishedReq': 'WidgetRequest' to generate when animation is complete.
 -}
-data FadeCfg e = FadeCfg {
-  _fdcAutoStart :: Maybe Bool,
-  _fdcDuration :: Maybe Millisecond,
-  _fdcOnFinished :: [e]
+data FadeCfg s e = FadeCfg {
```

Comment:
@Deltaspace0 could this one be made a `newtype`?

Reply:
I have converted it and `ZoomCfg` to a `newtype`.